docs/cvl/ghosts.md

```diff
@@ -194,6 +194,8 @@ Restrictions on ghost axioms
 Ghosts vs. persistent ghosts
 ----------------------------
 
+A `persistent ghost` is a `ghost` that will never be {ref}`havoc <glossary>`. The value of a non-persistent `ghost` will be `havoc'ed` when the Prover `havoc`s the storage, a `persistent ghost` however will keep its value when storage is havoced.
```

Comment:
We should also mention that persistent ghosts do not revert.

Reply:
@shoham-certora Do you mean persistent ghost _functions_ do not revert?